src/gui/tray/ActivityListModel.cpp

```diff
@@ -119,7 +114,7 @@ QVariant ActivityListModel::data(const QModelIndex &index, int role) const
                 }
             }
 
-            return QUrl::fromLocalFile(path);
+            return QUrl::fromLocalFile(list.at(0));
```

Comment:
list.first or constFirst if possible

Reply:
Not sure it makes a huge difference in that context, but yeah at least constFirst() conveys the intent better.